src/impl/util.js

```diff
@@ -96,10 +96,13 @@ export class Util {
   }
 
   static daysInMonth(year, month) {
-    if (month === 2) {
-      return Util.isLeapYear(year) ? 29 : 28;
+    const modMonth = 1 + (((month - 1) % 12) + 12) % 12,
```

Comment:
This seems overly complicated. In what circumstance is that different from:

```js
1 + (month - 1) % 12;
```

I think the more complicated one is actually broken for 0:

```js
modMonth = (month) => 1 + (((month - 1) % 12) + 12) % 12
modMonth2 = (month) => 1 + (month - 1) % 12
modMonth(0) //=> 12, seems wrong
modMonth2(0) //=> 0, seems right
```

I *think* they both work out for negatives (e.g. `modMonth(-25) //=> 11`), but I want to see some tests for minus.

Reply:
Since in Luxon, months are 1-indexed, 0  is not a valid month. January is 1, so 0 should be December the previous year. (12)

Also, the simplified version doesn't work with negatives. 
 
`modMonth2(-25) //=> - 1`

Reply:
I could refactor this expression into an `Util.positiveMod` or something like that, since I agree it looks more complicated than it needs to be. Some description of what this does would increase clarity.

Reply:
Ah, right, I got confused in the middle of that analysis. Cool.